user-guide/toc.yml

```diff
@@ -1945,6 +1934,80 @@
                 topicUid: DashboardNumericInputFeed
               - name: Text input
                 topicUid: DashboardTextInputFeed
+        - name: Generic Query Interface (GQI)
+          topicUid: Generic_Query_Interface
+          items:
+            - name: Creating a GQI query
+              topicUid: Creating_GQI_query
+            - name: Query data sources
+              topicUid: Query_data_sources
+            - name: Query operators
+              topicUid: Query_operators
+              items:
+                - name: Aggregate
+                  topicUid: GQI_Aggregate
+                - name: Column manipulations
+                  topicUid: GQI_Column_manipulations
+                - name: Filter
+                  topicUid: GQI_Filter
+                - name: Join
+                  topicUid: GQI_Join
+                - name: Select
+                  topicUid: GQI_Select
+                - name: Sort
+                  topicUid: GQI_Sort
+                - name: Sort by
+                  topicUid: GQI_Sort_by
+                - name: Then sort by
+                  topicUid: GQI_Then_sort_by
+                - name: Top X
+                  topicUid: GQI_Top_X
+            - name: Extensions
+              topicUid: Configuring_an_ad_hoc_data_source_in_a_query
+              items:
+                - name: Configuring an ad hoc data source
+                  topicUid: Configuring_an_ad_hoc_data_source_in_a_query
+                  items:
+                    - name: Building blocks of queries with ad hoc data
+                      topicUid: Ad_hoc_Building_blocks
+                    - name: Life cycle of queries with ad hoc data
+                      topicUid: Ad_hoc_Life_cycle
+                    - name: Tutorials
+                      topicUid: Ad_hoc_Tutorials
+                - name: Configuring a custom operator
+                  topicUid: GQI_Custom_Operator
+                  items:
+                    - name: Building blocks of a custom operator
+                      topicUid: CO_Building_blocks
+                    - name: Life cycle of a custom operator
+                      topicUid: CO_Life_cycle
```

Comment:
We already have some tutorials in the internal-docs. We can then move those here :)

Reply:
I will include these tutorials in a second revision of the GQI documentation. This will allow me to move and review them without delaying the publication of the restructured GQI documentation.